lib/widgets/fields/alignment_field.dart

```diff
@@ -0,0 +1,155 @@
+
+import 'package:flutter/material.dart';
+
+class AlignmentField extends StatefulWidget {
+  final void Function(Alignment) onChanged;
+
+  const AlignmentField({
+    Key? key,
+    required this.onChanged,
+  }) : super(key: key);
+
+  @override
+  _AlignmentFieldState createState() => _AlignmentFieldState();
+}
+
+class _AlignmentFieldState extends State<AlignmentField> {
+  int selectedOption = 5;
+  int onHoverOption = 5;
+  bool isHover = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          // height: 71,
+          width: 318,
+          decoration: BoxDecoration(
+            color: const Color(0xFF36343B),
+            borderRadius: BorderRadius.circular(8),
+            border: Border.all(
+              color: Colors.white,
+              width: 1,
+            ),
+          ),
+          child: Column(
```

Comment:
Use more optimal approach and less code repetiton,         


  child: Column(
            children: List<Widget>.generate(3, (row) {
              return Row(
                children: List<Widget>.generate(3, (col) {
                  int index = row * 3 + col + 1;
                  return customRadioButton(index);
                }),
              );
            }),
          ),